scene/gui/color_picker.h

```diff
@@ -317,6 +311,7 @@ class ColorPicker : public VBoxContainer {
 		int sv_height = 0;
 		int sv_width = 0;
 		int h_width = 0;
+		int colorize_sliders = 0;
```

Comment:
```suggestion
		bool colorize_sliders = false;
```
The underlying constant can be bool, even if it still shows as integer.

Reply:
You forgot `false`.